frontend/src/lib/k8s/api/v2/webSocket.ts

```diff
@@ -167,3 +169,93 @@ export function useWebSocket<T>({
     };
   }, [enabled, url, protocols, type, cluster]);
 }
+
+export type WebSocketConnectionRequest<T> = {
+  cluster: string;
+  url: string;
+  onMessage: (data: T) => void;
+};
+
+/**
+ * Creates or joins mutiple existing WebSocket connections
+ *
+ * @param url - endpoint URL
+ * @param options - WebSocket options
+ */
+export function useWebSockets<T>({
```

Comment:
could useWebSockets use the singular useWebSocket func but for multiple?

Reply:
good point. useWebSocket now uses useWebSockets